exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/TraceConfiguration.java

```diff
@@ -127,7 +136,7 @@ public abstract class TraceConfiguration {
    */
   public static Builder builder() {
     return new AutoValue_TraceConfiguration.Builder()
-        .setProjectId(DEFAULT_PROJECT_ID)
+        .setProjectId("")
```

Comment:
If you follow my suggestion earlier, this should become `.setProjectIdSupplier(ServiceOptions::getDefaultProjectId)`

Reply:
This would be an API change. I am assuming this would **replace** the current `setProjectId` ? Because exposing both setter would be confusing for the user.

Reply:
No I'm expecting this to be an additive change (not a backwards compatibility break). We can resolve the confusion by documenting the provider setter.